### PR TITLE
Settings view and startofday setting

### PR DESCRIPTION
**Pull request by johan-bjareholt**
Was surprisingly easy to implement

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -8,6 +8,15 @@ import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap-vue/dist/bootstrap-vue.css';
 Vue.use(BootstrapVue);
 
+// Load and setup vue-cookies
+import VueCookies from 'vue-cookies'
```
Comment (ErikBjare) [inline src/main.js]: You should probably use localstorage instead.

File: src/main.js [hunk 2]
```diff
@@ -8,6 +8,15 @@ import 'bootstrap/dist/css/bootstrap.css';
...
+Vue.use(VueCookies)
+// set default config
+VueCookies.config('aw-webui')
+if (!VueCookies.isKey('start-of-day')) {
+    VueCookies.set('start-of-day','02:00');
```
Comment (ErikBjare) [inline src/main.js]: I'd suggest a default of 04:00, that's what the Oura app uses (and some other apps I've used).

File: src/views/Settings.vue [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  h3 Settings
+
+  table.table
+    tbody
+      tr
+        td Start of day
+        td
+          input(type="time" :value="startOfDay" @change="setStartOfDay($event.target.value)")
+        td
+          | Sets the time where the start/end of a day is in the dayily activity view.
```
Comment (ErikBjare) [inline src/views/Settings.vue]: Misspelling of 'daily'

File: src/main.js (hunk 1 above)
Comment (johan-bjareholt) [reply src/main.js]: Fixed

File: src/main.js (hunk 2 above)
Comment (johan-bjareholt) [reply src/main.js]: Fixed

File: src/views/Settings.vue (hunk 3 above)
Comment (johan-bjareholt) [reply src/views/Settings.vue]: Fixed

**Pull request merged**